src/hotspot/share/prims/jvmtiRawMonitor.hpp

```diff
@@ -111,7 +111,7 @@ class JvmtiRawMonitor : public CHeapObj<mtSynchronizer>  {
   };
 
   // Non-aborting operator new
-  void* operator new(size_t size) throw() {
+  void* operator new(size_t size) {
```

Comment:
This change is incorrect, as this can quite obviously return null.  And that seems to be intentional.
Presumably the callers are checking for a possible null allocation result (else there is a bug).  I think
it would be less confusing if this took a `std::nothrow_t` to be explicit about it's behavior, and updated
the caller(s) accordingly.  That would match the usual idiom.